lib/Doctrine/DBAL/Connection.php

```diff
@@ -489,12 +489,14 @@ public function setAutoCommit($autoCommit)
      * Sets the fetch mode.
      *
      * @param integer $fetchMode
-     *
+     * @param mixed $arg2
```

Comment:
Please rename them accordingly for clarity. Tests are also required to  clarify the use-case scenario (and validate it)